main.nf

```diff
@@ -58,11 +58,16 @@ def helpMessage() {
 
     Other options:
       --outdir                      The output directory where the results will be saved
+      --workDir                     The temporary directory where intermediate data will be saved
```

Comment:
Nextflow already has a core function for this:

```
    -w, -work-dir
       Directory where intermediate result files are stored
```

Reply:
Took care of using that one in https://github.com/nf-core/rnaseq/pull/42/commits/70d4bb0975318f54fdbff6be1ad13b39bd69804a